### ISSUE TITLE
[BUG] Incompatibility with Seamless Loading Screen

### ISSUE THREAD

**Issue by SylisMC** [juancarloscp52/BedrockIfy#45]
Incompatibility with [Seamless Loading Screen](https://www.curseforge.com/minecraft/mc-mods/seamless-loading-screen)
Setting client core overlay injection to `false` helps the issue, but there is still a jarring flash between loading the world.
![error](https://user-images.githubusercontent.com/19602662/104078023-04396700-51ea-11eb-959e-d88263bd23d3.gif)

**Comment by juancarloscp52** [comment]
Hi! Try disabling the option "Enable Bedrock Loading Screens" under the "GUI/Visual Improvements" section of bedrockIfy settings. If the panorama background is still showing you might need to also disable the panorama background on all screens option

**Comment by SylisMC** [comment]
I've tried turning every option in the config off, That works... but the loading bar is so much nicer than the chunk view garbage Mojang left us, if there is a way, and you are wiling, it would be nice to have the best of both worlds.

**Comment by juancarloscp52** [comment]
Hmm I'll take a look at what can I do, although it might take some time to make it onto a release.

**Comment by SylisMC** [comment]
The owner of SLS is also trying to work on this, you may want to link up @Minenash
https://github.com/Minenash/Seamless-Loading-Screen/issues/9

**Comment by Minenash** [comment]
After finally getting around to figuring out how to get do the modImplemntation for this mod, my mixin idea worked pretty well:
```java
@Mixin(value = LoadingScreenWidget.class, remap = false)
public class BedrockifyMixin {

    @Inject(method = "render", at = @At("HEAD"))
    private void renderBackground(MatrixStack matrices, int _width, int _height, Text _title, Text _message, int _progress, CallbackInfo _info) {
        if (ScreenshotLoader.loaded)
            ScreenshotLoader.render(MinecraftClient.getInstance().currentScreen, matrices);
    }

}
```
The only problem is that Bedrockify's hud overlay doesn't respect `client.options.hudhidden`, so it shows up in the screenshot. This can be fixed by adding ` && !client.options.hudHidden` to [this line](https://github.com/juancarloscp52/BedrockIfy/blob/4438adf8e1a453a95e460cb110c412f5845071c1/src/main/java/me/juancarloscp52/bedrockify/client/gui/Overlay.java#L38). It is kinda weird that it respects F1, even though the only thing that I thought it did was set `client.options.hudhidden` ¯\\_ (ツ)_/¯

**Comment by juancarloscp52** [comment]
Awesome!! Thanks!! Yeah I forgot that you could hide the HUD with F1 when adding the hide option for F3. Thanks for your fix and your time! I'll add the change to the source ASAP although the release of a new version might take some time.

**Comment by Minenash** [comment]
The funny thing is that Bedrockify *is* hidden with F1 (or at the very least the cords)

**Comment by juancarloscp52** [comment]
The fix has been released with the latest bedrockIfy version (0.2.2)!
